### PR TITLE
Remove explicit passwords from documentation

### PR DESCRIPTION
**Pull request by russgold**
The existing documentation listed some passwords explicitly. This change removes them.

### CODE REVIEW

File: samples/kubernetes/end2end/docs/existing-domain.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ First, get the wls-exporter WAR file following the [Downloading the release](htt
 After the wls-exporter is installed successfully, you can check the exported WebLogic runtime metrics from each server. See the detail guide [here](04-wls-domain.md#check-the-weblogic-runtime-metrics).
 
 ### Updating Prometheus Configuration
-Next, we need to inform Prometheus server to scrape metric from the WebLogic domain. Basically, we need to add a new job to `prometheus.yml`, the configuration file of the Prometheus server. The job is similar to what we added for `domain1` in [values.yaml](../prometheus/values.yaml#L59) of the Prometheus chart. As shown below, lines in red need to be updated properly.
+Next, we need to configure Prometheus to scrape metrics from the WebLogic domain. The sample value Basically, we need to add a new job to `prometheus.yml`, the configuration file of the Prometheus server. The job is similar to what we added for `domain1` in [values.yaml](../prometheus/values.yaml#L59) of the Prometheus chart. As shown below, lines in red need to be updated properly.
```
Comment (marinakog) [inline samples/kubernetes/end2end/docs/existing-domain.md:14]: The sample value basically,

Comment (marinakog) [reply samples/kubernetes/end2end/docs/existing-domain.md:14]: maybe reword?

File: samples/kubernetes/end2end/prometheus/values.yaml [hunk 2]
```diff
@@ -84,5 +84,5 @@ extraScrapeConfigs: |
         action: replace
         target_label: pod_name
       basic_auth:
-        username: weblogic
-        password: welcome1
+        username: <specify a user>
+        password: <specify a password>
```
Comment (marinakog) [inline samples/kubernetes/end2end/prometheus/values.yaml:88]: specify a valid WebLogic user

File: samples/kubernetes/deployments/prometheus-deployment.yaml [hunk 3]
```diff
@@ -252,5 +252,5 @@ data:
           target_label: webapp
 
         basic_auth:
-          username: weblogic
-          password: welcome1
+          username: <a username>
```
Comment (marinakog) [inline samples/kubernetes/deployments/prometheus-deployment.yaml:255]: a weblogic username ?

File: samples/kubernetes/deployments/prometheus-withalertmanager-deployment.yaml [hunk 4]
```diff
@@ -252,5 +252,5 @@ data:
           target_label: webapp
 
         basic_auth:
-          username: weblogic
-          password: welcome1
+          username: <a username>
```
Comment (marinakog) [inline samples/kubernetes/deployments/prometheus-withalertmanager-deployment.yaml:255]: a weblogic username

File: samples/kubernetes/end2end/prometheus/values.yaml (hunk 2 above)
Comment (marinakog) [reply samples/kubernetes/end2end/prometheus/values.yaml:88]: or specify a weblogic user ?

**Review by marinakog — CHANGES_REQUESTED** [review]

File: samples/kubernetes/end2end/docs/existing-domain.md (hunk 1 above)
Comment (russgold) [reply samples/kubernetes/end2end/docs/existing-domain.md:14]: The entire section needs to be redone. Derek is filing a new JIRA for it.

Comment (russgold) [reply samples/kubernetes/end2end/docs/existing-domain.md:14]: I will actually revert the rewrite I started above. A partial rewrite is not helpful.

**Comment by russgold** [comment]
Monica has withdrawn the JIRA, assuming we can fix it when we revisit the overall docs
